en/sql/datatype.rst

```diff
@@ -1179,10 +1179,10 @@ When the length of a character string exceeds *n*, they are truncated. When char
 
 ::
 
-    If you specify 'pacesetter' as CHAR(4), 'pace' is stored (truncated as the length of the character string is greater than 4).
+    If you specify 'pacesetter' as CHAR(4), 'pace' may be stored or error occurs at the configuration value of allow_truncated_string (truncated as the length of the character string is greater than 4).
```

Comment:
at the configuration value of allow_truncated_string
==> depending on the configuration value of allow_truncated_string